### PR TITLE
xinput: add list-props and set-prop examples

### PR DESCRIPTION
**Pull request by Ben-D-Anderson**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Hi, thanks for your commit!
From what I see, this should be committed and merged probably within 2 days

File: pages/linux/xinput.md [hunk 1]
```diff
@@ -21,3 +21,11 @@
 - Reattach an input as slave to a master:
 
 `xinput reattach {{id}} {{master_id}}`
+
+- List settings of an input:
```
Comment (CleanMachine1) [inline pages/linux/xinput.md:25]:
```suggestion
- List settings of an input device:
```

File: pages/linux/xinput.md [hunk 2]
```diff
@@ -21,3 +21,11 @@
...
+
+`xinput list-props {{id}}`
+
+- Change setting of an input:
```
Comment (CleanMachine1) [inline pages/linux/xinput.md:29]:
```suggestion
- Change a setting of an input device:
```

File: pages/linux/xinput.md [hunk 3]
```diff
@@ -21,3 +21,11 @@
...
+
+`xinput set-prop "{{id}}" "{{setting_id}}" "{{value}}"`
```
Comment (navarroaxel) [inline pages/linux/xinput.md:31]: Are the quotes needed for the possible `id` values?

Comment (Ben-D-Anderson) [reply pages/linux/xinput.md:31]: Apologies, I previously believed they were but I just tested it without and it works fine.

**Review by CleanMachine1 — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Ben-D-Anderson!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**